### DISCUSSION TITLE
Setting `Input` cursor color?

### DISCUSSION THREAD

**Discussion by GollyJer** [GeekyAnts/NativeBase#5549] [Q&A] [answered]
I'm trying to set the cursor color of the `Input` component but it only displays correctly for a split second before it is overwritten.

I've tried 
```javascript 
cursorColor: 'primary.500',
selectionColor: 'primary.500',
```

In this example the color is orange. It only shows very quickly.
What is the right approach to keep the desired cursor color? Thanks!

https://user-images.githubusercontent.com/689204/201429008-1163f225-8b68-4c4f-913c-271a35275db4.mp4

**Comment by ankit-tailor** [comment, marked as answer]
Hey @GollyJer, Thanks for reporting the issue. There is already an open issue https://github.com/GeekyAnts/NativeBase/issues/5420. We'll fix it.

**Reply by GollyJer** [reply]
Sounds good. I didn't find that one because it's related to `selectionColor` and this is about `cursorColor`.
I'll be sure to test this in the next release. Thanks!
